bindings/experimental/distrdf/python/DistRDF/Backends/__init__.py

```diff
@@ -0,0 +1,9 @@
+import importlib
+import pkgutil
+import DistRDF
+
+for _, module_name, _ in pkgutil.walk_packages(__path__):
+    module = importlib.import_module(__name__ + "." + module_name)
+    setattr(DistRDF, module_name, module)
```

Comment:
To be decided here:
1. We leave the injection as it is, which means `DistRDF` is injected as `ROOT.RDF.Distributed`.
2. We do the injection here in a new module called `ROOT.RDF.Distributed` (advantage: it would have the right name and not the internal one when being checked by a user).
3. We leave the responsibility of the injection to the backends.